windows: Fix the event loop to wake on accessibility requests (#4058)
rust-windowing/winit: src/changelog/unreleased.md

- On macOS, fixed the scancode conversion for audio volume keys.
- On macOS, fixed the scancode conversion for `IntlBackslash`.
- On macOS, fixed redundant `SurfaceResized` event at window creation.

> - On Windows, fixed the event loop to wake on accessibility requests.

Reviewer: ```suggestion
- On Windows, fixed the event loop not waking on accessibility requests.
```
